src/api/environment.cc

```diff
@@ -703,9 +702,10 @@ Maybe<bool> InitializePrimordials(Local<Context> context) {
                                         "internal/per_context/messageport",
                                         nullptr};
 
+  auto builtin_loader = builtins::BuiltinLoader::Create();
```

Comment:
Hmm, this doesn't look right, then we'll create two builtin loaders to initialize an Environment without snapshot? Can we make it part of the isolate data instead?

Reply:
We could do that, but none of our external APIs for context creation really support taking in an `IsolateData*` … we could add this, but with the updated code now, I think the performance impact here is not something we need to worry about.

Reply:
Or can it be `thread_local`? Assuming we are going to support multiple isolates per thread?

Reply:
Yeah, `thread_local` should be fine :+1: added that and a comment about why this should be okay here